docs/inventory-management.md

```diff
@@ -11,55 +11,92 @@ the mapping of the machine to it's configuration and assigned cluster.
 
 ### MachineInventory
 
+The `MachineInventory` holds all the relevant information for a registered machine.  
+Upon successful registration, the `MachineInventory` will inherit all `machineInventoryLabels` defined in the related `MachineRegistration`.  
+Additionally, the machine `annotations` will also be updated on each successful registration.  
+
+By default, Elemental Teal machines will attempt a registration update every 24 hours to update labels and annotations.  
```

Comment:
It does attempt registration at boot, but most likely will be skipped due to the 24 hours timer on the last successful updated. 
I omitted it to not mislead users who will otherwise expect an update every boot.
I think we can address it here: https://github.com/rancher/elemental-operator/issues/481